netty/src/main/java/io/grpc/netty/ProtocolNegotiators.java

```diff
@@ -515,6 +527,11 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
       }
       super.userEventTriggered(ctx, evt);
     }
+
+    private SslHandler sslHandler(ChannelPipeline pipeline) {
```

Comment:
Do you mean ServerTlsHandler.sslHandler()? It does have two callers. Granted it could also be reduced to a local variable.